src/components/App/AppSagas.js

```diff
@@ -27,7 +27,14 @@ export function* loadApp() {
   yield showSpinner('Loading Google Api');
   yield googleApiLoader.load();
   yield googleDriveApi.loadDriveApi();
-  const rootNote = yield noteStorage.scanDrive();
+  let rootNote;
+  const lastViewedNoteId = localStorage.getItem('lastViewedNoteId');
+  if (lastViewedNoteId) {
+    rootNote = yield googleDriveApi.findNotesById(lastViewedNoteId);
```

Comment:
you call `findNotes...` but assign it not to an array of notes, but to one note: `rootNote`.
So should it be `findNote...` ? (without `S`)

Reply:
Also, it's not `rootNote`. It's last viewed note :) Root note is `NeuralNotes`.